examples/platform/silabs/uart.cpp

```diff
@@ -413,7 +413,7 @@ static void UART_rx_callback(UARTDRV_Handle_t handle, Ecode_t transferStatus, ui
 
 #ifdef ENABLE_CHIP_SHELL
     chip::NotifyShellProcess();
-#elif !defined(PW_RPC_ENABLED)
+#elif !defined(PW_RPC_ENABLED) && !defined(SL_WIFI)
```

Comment:
Should we instead define gate with `CHIP_DEVICE_CONFIG_THREAD_ENABLE_CLI` since the Ot-Cli should not be enabled?

Reply:
Perhaps. But not in this PR which is a hotfix to unblock the release build. The SL_WIFI macro is already used in other places in this file.

Reply:
I don't see the relevance with a hotfix but i won't block the PR. Can you file a follow up item when the upstream PR is open?

Reply:
MATTER-4295 has been created to follow up